Easydict.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -335,5 +335,5 @@
       }
     }
   ],
-  "version" : 2
+  "version" : 3
```

Comment:
Xcode may sometimes automatically modify this value, which can lead to CI failures, like in https://github.com/tisfeng/Easydict/actions/runs/9928827058/job/27425612841?pr=613 and https://github.com/tisfeng/Easydict/pull/473#issuecomment-2027908492.

You can try updating the version of Xcode in `codeql.yml`.

Reply:
i modify the value to 2